pyspark_ai/prompt.py

```diff
@@ -205,6 +216,22 @@
     prefix=SPARK_SQL_PREFIX,
 )
 
+SQL_CHAIN_EXAMPLES = [
+    sql_question1 + f"\nAnswer:\n```{sql_answer1}```",
+    sql_question2 + f"\nAnswer:\n```{sql_answer2}```",
```

Comment:
do we want to give it all 4 examples?

Reply:
I am thinking about removing example 4 in all the agent/chains.